backend/src/bundles/user-details/user-details.controller.ts

```diff
@@ -0,0 +1,326 @@
+     *          companyLogoId:
+     *            type: string
+     *          companyWebsite:
+     *            type: string
+     *          employerPosition:
+     *            type: string
+     *          cvId:
+     *            type: string
+     *   securitySchemes:
+     *     bearerAuth: # Define the JWT security scheme
+     *       type: http
+     *       scheme: bearer
+     *       bearerFormat: JWT
+     */
+    private async update(
+        options: ApiHandlerOptions<{
+            body: UserDetailsUpdateRequestDto;
+        }>,
+    ): Promise<ApiHandlerResponse> {
+        return {
+            status: HttpCode.OK,
+            payload: await this.userDetailsService.update(options.body),
+        };
+    }
+
+    /**
+     * @swagger
+     * /user-details/approve:
+     *    patch:
+     *      tags:
+     *        - User Details
+     *      description: Approves a user's details
+     *      security:
+     *        - bearerAuth: []
+     *      requestBody:
+     *        description: User detail update object
+     *        required: true
+     *        content:
+     *          application/json:
+     *            schema:
+     *              type: object
+     *              $ref: '#/components/schemas/UserDetailsApproveRequestDto'
+     *            examples:
+     *              example-approve:
+     *                value:
+     *                  id: '550e8400-e29b-41d4-a716-446655440000'
+     *                  isApproved: true
+     *              example-denied:
+     *                value:
+     *                  id: '5a4b4ee2-7089-4c27-88d0-9e5e60ccf0dd'
+     *                  isApproved: false
+     *                  deniedReason: 'Incomplete information'
+     *      responses:
+     *         200:
+     *           description: Successful operation
+     *           content:
+     *             application/json:
+     *               schema:
+     *                 type: object
+     *                 $ref: '#/components/schemas/UserDetails'
+     * components:
+     *   schemas:
+     *      UserDetailsApproveRequestDto:
+     *        type: object
+     *        properties:
+     *          id:
+     *            format: uuid #Example: '550e8400-e29b-41d4-a716-446655440000'
+     *            type: string
+     *            required: true
+     *          isApproved:
+     *            type: boolean
+     *            required: true
+     *          deniedReason:
+     *            type: string
+     *   securitySchemes:
+     *     bearerAuth: # Define the JWT security scheme
+     *       type: http
+     *       scheme: bearer
+     *       bearerFormat: JWT
+     */
+    private async approve(
+        options: ApiHandlerOptions<{
+            body: UserDetailsApproveRequestDto;
+        }>,
+    ): Promise<ApiHandlerResponse> {
+        return {
+            status: HttpCode.OK,
+            payload: await this.userDetailsService.approve(options.body),
+        };
+    }
+}
```

Comment:
You don't need this in this PR. This is another task. Please don't go beyond the scope of your task !

Reply:
Can I leave it in this task? Because it has already implemented.